src/github.com/mozilla-services/pushgo/simplepush/emcee_store.go

```diff
@@ -640,9 +646,9 @@ func (s *EmceeStore) DropAll(uaid string) error {
 	}
 	defer s.releaseWithout(client, &err)
 	for _, chid := range chids {
-		key, ok := s.IDsToKey(uaid, chid)
-		if !ok {
-			return ErrInvalidKey
+		key, err := s.IDsToKey(uaid, chid)
+		if err != nil {
```

Comment:
Related to above, apparently an error in IDsToKey is bad enough to return it here, but above its silently ignored.